### ISSUE TITLE
Default populate malware_config

### ISSUE THREAD

**Issue by andr6** [certsocietegenerale/fame#16]
Would it be possible to provide some default values  for this configuration ?

**Comment by gaelmuller** [comment]
No, I don't think there is nothing generic that we could provide. What are your expecting ?

**Comment by andr6** [comment]
@gaelmuller  . Maybe providing some examples could be good.

**Comment by cvdsouza** [comment]
Hi, just wondering if an example of what the malware_config patterns would look like can be provided ? The user guide doesn't have any info on it.

**Comment by functure** [comment]
Hi! It would be nice to add either a brief explanation in tutorial or an example configuration
